charts/kube-prometheus-stack/Chart.yaml

```
@@ -23,7 +23,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 56.7.0
+version: 56.7.1
```

This is more a feature than a fix, so we should bump minor:
```suggestion
version: 56.8.0
```

---

bumped up to 56.9.0